static/app/components/autoComplete.tsx

```diff
@@ -31,19 +31,18 @@ const defaultProps = {
   shouldSelectWithTab: false,
 };
 
-type GetInputArgs = {
+type GetInputArgs<E extends HTMLInputElement> = {
```

Comment:
I needed to make these generic to work around the fact that react's KeyboardEvent has become more tight as to interoperability between Element types.